dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -31,7 +31,54 @@
     ) as {{ col }}
 {% endmacro %}
 
-{% macro athena__create_csv_table(model, agate_table) %}
+{% macro create_csv_table_insert(model, agate_table) %}
+  {%- set identifier = model['alias'] -%}
+
+  {%- set lf_tags_config = config.get('lf_tags_config') -%}
+  {%- set lf_grants = config.get('lf_grants') -%}
+  {%- set column_override = config.get('column_types', {}) -%}
+  {%- set quote_seed_column = config.get('quote_columns') -%}
+  {%- set s3_data_dir = config.get('s3_data_dir', target.s3_data_dir) -%}
+  {%- set s3_data_naming = config.get('s3_data_naming', target.s3_data_naming) -%}
+  {%- set external_location = config.get('external_location') -%}
+
+  {%- set relation = api.Relation.create(
+    identifier=identifier,
+    schema=model.schema,
+    database=model.database,
+    type='table'
+  ) -%}
+
+  {%- set location = adapter.generate_s3_location(relation,
+                                                 s3_data_dir,
+                                                 s3_data_naming,
+                                                 external_location,
+                                                 temporary) -%}
+
+  {% set sql_table %}
+    create external table {{ relation.render_hive() }} (
+        {%- for col_name in agate_table.column_names -%}
+          {%- set inferred_type = adapter.convert_type(agate_table, loop.index0) -%}
+          {%- set type = column_override.get(col_name, inferred_type) -%}
+          {%- set type = type if type != "string" else "varchar" -%}
+          {%- set column_name = (col_name | string) -%}
+          {{ adapter.quote_seed_column(column_name, quote_seed_column) }} {{ ddl_data_type(type) }} {%- if not loop.last -%}, {% endif -%}
+        {%- endfor -%}
+    )
+    location '{{ location }}'
+
+  {% endset %}
+
+  {% call statement('_') -%}
+    {{ sql_table }}
+  {%- endcall %}
+
+
```

Comment:
```suggestion

  {% if lf_tags_config is not none %}
    {{ adapter.add_lf_tags(relation, lf_tags_config) }}
  {% endif %}

  {% if lf_grants is not none %}
    {{ adapter.apply_lf_grants(relation, lf_grants) }}
  {% endif %}

```
seems we forgot to apply lf_tags and lf_grants for this case